.github/workflows/bot-all-cards-csharp.yml

```diff
@@ -0,0 +1,70 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app (Bot-All-Cards) to Azure Web App
```

Comment:
Add demo manifest.

Need to prepare with deployment environment.

Reply:
Fixed